pkg/util/validation/formats.go

```diff
@@ -287,7 +285,9 @@ func (FormatLDAPSearchFilterTemplate) CheckFormat(value interface{}) error {
 		return nil
 	}
 
-	tmpl, err := template.New("search_filter").Parse(str)
+	str = strings.TrimSpace(str)
+
+	tmpl, err := template.New("search_filter").Parse(strings.Trim(str, "\n"))
```

Comment:
I think you do not need to trim space here, you just need to trim space on the rendered result.